src/main/java/com/google/sps/language/Analyze.java

```diff
@@ -0,0 +1,325 @@
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.sps;
+
+import com.google.cloud.language.v1.AnalyzeEntitiesRequest;
+import com.google.cloud.language.v1.AnalyzeEntitiesResponse;
+import com.google.cloud.language.v1.AnalyzeEntitySentimentRequest;
+import com.google.cloud.language.v1.AnalyzeEntitySentimentResponse;
+import com.google.cloud.language.v1.AnalyzeSentimentResponse;
+import com.google.cloud.language.v1.AnalyzeSyntaxRequest;
+import com.google.cloud.language.v1.AnalyzeSyntaxResponse;
+import com.google.cloud.language.v1.ClassificationCategory;
+import com.google.cloud.language.v1.ClassifyTextRequest;
+import com.google.cloud.language.v1.ClassifyTextResponse;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.Document.Type;
+import com.google.cloud.language.v1.EncodingType;
+import com.google.cloud.language.v1.Entity;
+import com.google.cloud.language.v1.EntityMention;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import com.google.cloud.language.v1.Token;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Uses the Natural Language API to perform entity, sentiment and syntax
+ * analysis.
+ */
+public class Analyze {  
+    
+    /** Run sentiment analysis and return sentiment score */
+  public float getSentimentScore(String message) throws Exception {
+    Document doc =
+        Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float score = sentiment.getScore();
+    // float score = Math.round(sentiment.getScore() * 100.0) / 100.0;
+    languageService.close();
+    return score;
+  }
+
+  /** Identifies entities in a given string. */
+  public static void analyzeEntitiesText(String text) throws Exception {
+
+    // Instantiate the Language client com.google.cloud.language.v1.LanguageServiceClient
+    try (LanguageServiceClient language = LanguageServiceClient.create()) {
+      Document doc = Document.newBuilder().setContent(text).setType(Type.PLAIN_TEXT).build();
+      AnalyzeEntitiesRequest request =
+          AnalyzeEntitiesRequest.newBuilder()
+              .setDocument(doc)
+              .setEncodingType(EncodingType.UTF16)
+              .build();
+
+      AnalyzeEntitiesResponse response = language.analyzeEntities(request);
+
+      // Print the response
+      for (Entity entity : response.getEntitiesList()) {
```

Comment:
If you plan on keeping this, extract code into separate method to use on line 105 as well.